From d50999669e43013d6e93a550a6ca949db1603375 Mon Sep 17 00:00:00 2001
From: Ecco park <eccopark@broadcom.com>
Date: Wed, 13 Aug 2014 12:19:12 -0700
Subject: [PATCH 52/77] net: wireless: bcmdhd fixed regression of SOFTAP
 throughput

Increase TXGLOM_SIZE and sg_list entries exceed limit issue fix.

b/17007818 Throughput regression of SOFTAP
Change-Id: I4273a6ab08963ec6808539e772627e1eb990f06c
Signed-off-by: Iliyan Malchev <malchev@google.com>
Signed-off-by: Ecco park <eccopark@broadcom.com>
---
 drivers/net/wireless/bcmdhd/include/bcmsdh_sdmmc.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/wireless/bcmdhd/include/bcmsdh_sdmmc.h b/drivers/net/wireless/bcmdhd/include/bcmsdh_sdmmc.h
index 69d8e7a8cae4..af265df5bbee 100644
--- a/drivers/net/wireless/bcmdhd/include/bcmsdh_sdmmc.h
+++ b/drivers/net/wireless/bcmdhd/include/bcmsdh_sdmmc.h
@@ -57,7 +57,7 @@
 /* private bus modes */
 #define SDIOH_MODE_SD4		2
 #define CLIENT_INTR			0x100	/* Get rid of this! */
-#define SDIOH_SDMMC_MAX_SG_ENTRIES	32
+#define SDIOH_SDMMC_MAX_SG_ENTRIES	(SDPCM_MAXGLOM_SIZE+2)
 
 struct sdioh_info {
 	osl_t		*osh;			/* osh handler */
-- 
2.0.3

